pages/exercises/[lessonSlug].tsx

```diff
@@ -25,12 +26,48 @@ const Exercises: React.FC<QueryDataProps<GetAppQuery>> = ({ queryData }) => {
 
   return (
     <Layout title={currentLesson.title}>
+      <ExercisesTitleCard
+        lessonDocUrl={currentLesson.docUrl}
+        lessonSlug={slug}
+      />
       <h1>{currentLesson.title}</h1>
       {alerts && <AlertsDisplay alerts={alerts} />}
     </Layout>
   )
 }
 
+type ExercisesTitleCardProps = {
+  lessonDocUrl: string | null | undefined
+  lessonSlug: string
+}
+
+const ExercisesTitleCard = ({
+  lessonDocUrl,
+  lessonSlug
+}: ExercisesTitleCardProps) => {
+  return (
+    <div className="card shadow-sm d-inline-flex flex-row p-2 mb-4">
+      {lessonDocUrl && (
+        <NavLink
+          path={lessonDocUrl}
+          className="py-2 px-3 mx-1 fw-bold text-primary"
+        >
+          LESSONS
+        </NavLink>
+      )}
+      <NavLink
+        path={`/curriculum/${lessonSlug}`}
+        className="py-2 px-3 mx-1 fw-bold text-primary"
+      >
+        CHALLENGES
+      </NavLink>
+      <div className="py-2 px-3 mx-1 fw-bold bg-primary text-white rounded">
+        EXERCISES
+      </div>
+    </div>
+  )
+}
```

Comment:
Right now, the nav card is really simple and it's only being used in this file so I decided to keep it in this file instead of making it a separate component. It looks similar to the AppNav component (https://github.com/garageScript/c0d3-app/blob/abede305660455eddd75c1877893fee6853ddeae/components/AppNav/AppNav.tsx), so maybe we should move it into the components/AppNav folder and share some styling.

Reply:
Can we make [AdminLessonNav](https://github.com/garageScript/c0d3-app/blob/master/components/admin/lessons/AdminLessonSideNavLayout.tsx) generic and use it here?

Reply:
That's a good idea. I'll make a separate issue and pull request which will make AdminLessonNav generic and easier to use. Once that pull request is merged, I'll use that generic component in this pull request.

Reply:
Here's the issue that was created: https://github.com/garageScript/c0d3-app/issues/2268